benderillo/strapi
# Pull latest Strapi in

### What does it do?
Pulling upstream strapi repo

### Why is it needed?

To stay up-to-date with upstream Strapi repo

### How to test it?

I don't know, I guess make sure your server doesn't crash and burn afterwards.
I wish I had some tests to run...

### Related issue(s)/PR(s)

N/A
